Documentation-versions.html

```diff
@@ -0,0 +1,143 @@
+                    <a class="btn btn-primary" id="extending.pdf">pdf &raquo;</a>
+                </div>
+                <!-- pending completion of DDF-1910 -->
+<!--                <div class="col-md-3">
+                        <a class="btn btn-primary" id=javadoc.html">Javadoc &raquo;</a>
+                </div> -->
+            </div>
+
+            <div class="spacer"></div>
+            <h1>Previous Versions</h1>
+
+            <div class="container col-md-8">
+                <table class="table table-striped table-hover">
+                    <thead>
+                        <tr>
+                            <th class="col-md-2">Version</th>
+                            <th class="col-md-2">Release Date</th>
+                            <th class="col-md-4">html</th>
+                            <th class="col-md-4">pdf</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <tr>
+                            <td>2.8.1</td>
+                            <td>12/21/2015</td>
+                            <td><a href="http://artifacts.codice.org//service/local/artifact/maven/redirect?g=ddf&a=docs&v=2.8.1&r=public&c=DDFOmnibus&e=html">html</a></td>
+                            <td><a href="http://artifacts.codice.org//service/local/artifact/maven/content?g=ddf&a=docs&v=2.8.1&r=public&c=DDFOmnibus&e=pdf">pdf</a></td>
+                        </tr>
+                        <tr>
+                            <td>2.8.0</td>
+                            <td>10/23/2015</td>
+                            <td><a href="http://artifacts.codice.org//service/local/artifact/maven/redirect?g=ddf&a=docs&v=2.8.0&r=public&c=DDFOmnibus&e=html">html</a></td>
+                            <td><a href="http://artifacts.codice.org//service/local/artifact/maven/content?g=ddf&a=docs&v=2.8.0&r=public&c=DDFOmnibus&e=pdf">pdf</a></td>
+                        </tr>
+                        <tr>
+                            <td>2.7.1</td>
+                            <td>9/9/2015</td>
+                            <td colspan="2"><a href="http://artifacts.codice.org//service/local/artifact/maven/redirect?g=ddf&a=docs&v=2.7.1&r=public&c=Export&e=zip">zip</a></td>
+                        </tr>
+                        <tr>
+                            <td>2.7.0</td>
+                            <td>7/13/2015</td>
+                            <td colspan="2"><a href="http://artifacts.codice.org//service/local/artifact/maven/redirect?g=ddf&a=docs&v=2.7.0&r=public&c=Export&e=zip">zip</a></td>
+                        </tr>
+                    </tbody>
+                </table>
+            </div>
+
+            <footer>
+                <div class="container">
+                    <div class="row">
+                        <div class="col-md-10">Website and software - <a href="http://www.codice.org" target="_blank">The Codice Foundation</a></div>

```

Comment:
Not an actual problem, but for some reason I feel like the 's' in software should be capitalized. Either way -> Not a bug.